crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -224,6 +227,25 @@ impl ParsedQProperty {
             }
         })
     }
+
+    /// Generates a list of method names which will be generated for this property
+    pub fn pending_methods(&self) -> Vec<Name> {
+        let mut pending = vec![];
+
+        if let FlagState::Auto = &self.flags.read {
+            pending.push(getter_name_from_property(&self.name));
+        }
+
+        if let Some(FlagState::Auto) = &self.flags.write {
+            pending.push(setter_name_from_property(&self.name));
+        }
+
+        if let Some(FlagState::Auto) = &self.flags.notify {
+            pending.push(notify_name_from_property(&self.name));
+        }
```

Comment:
This is a signal, not a method, so needs to be treated differently during lookup.
We wouldn't want someone to e.g. use:

```
#[qproperty(i32, number, READ, NOTIFY=number)]
```
Which cxx-qt-gen would now accept.

Reply:
Looks like this is resolved now.